<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Surface3DDock</class>
 <widget class="QWidget" name="Surface3DDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>307</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabGeneral">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="5" column="0">
        <widget class="QLabel" name="labelFile">
         <property name="text">
          <string>File</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="labelZ">
         <property name="text">
          <string>Z</string>
         </property>
        </widget>
       </item>
       <item row="8" column="3">
        <widget class="TreeViewComboBox" name="cbXCoordinate"/>
       </item>
       <item row="10" column="3">
        <widget class="TreeViewComboBox" name="cbZCoordinate"/>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QLabel" name="lPlotType">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Data:</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="labelY">
         <property name="text">
          <string>Y</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer_10">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>28</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lName">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="labelX">
         <property name="text">
          <string>X</string>
         </property>
        </widget>
       </item>
       <item row="9" column="3">
        <widget class="TreeViewComboBox" name="cbYCoordinate"/>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QLabel" name="labelSource">
         <property name="text">
          <string>Data source</string>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="KComboBox" name="cbDataSource"/>
       </item>
       <item row="12" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>159</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="11" column="3">
        <widget class="TreeViewComboBox" name="cbMatrix"/>
       </item>
       <item row="11" column="0" colspan="2">
        <widget class="QLabel" name="labelMatrix">
         <property name="text">
          <string>Matrix</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="KLineEdit" name="leComment">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>23</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="5" column="3">
        <widget class="KUrlRequester" name="cbFileRequester">
         <property name="filter">
          <string>*.obj *.stl</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="lComment">
         <property name="text">
          <string>Comment</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="KLineEdit" name="leName"/>
       </item>
       <item row="13" column="0" colspan="2">
        <widget class="QCheckBox" name="chkVisible">
         <property name="text">
          <string>visible</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="labelPointHeader">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Point</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>28</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabColorFilling">
      <attribute name="title">
       <string>Color Filling</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="lColorFillingType">
         <property name="text">
          <string>Type</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>13</width>
           <height>23</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2">
        <widget class="QComboBox" name="cbColorFillingType">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lColorFilling">
         <property name="text">
          <string>Color</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="KColorButton" name="kcbColorFilling"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lColorFillingMap">
         <property name="text">
          <string>Color map</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QComboBox" name="cbColorFillingMap"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lColorFillingMatrix">
         <property name="text">
          <string>Matrix</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="TreeViewComboBox" name="cbColorFillingMatrix"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lColorFillingOpacity">
         <property name="text">
          <string>Opacity</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QSpinBox" name="sbColorFillingOpacity">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>The opacity ranges from 0 to 100, where 0 is fully transparent and 100 is fully opaque.</string>
         </property>
         <property name="suffix">
          <string> %</string>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="singleStep">
          <number>10</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <spacer name="verticalSpacer_8">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>24</width>
           <height>352</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabMesh">
      <attribute name="title">
       <string>Mesh</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="0">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>28</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="labelType">
         <property name="text">
          <string>Type</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>13</width>
           <height>23</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2">
        <widget class="KComboBox" name="cbType">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="labelNode1">
         <property name="text">
          <string>Node 1</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="TreeViewComboBox" name="cbNode1"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="labelNode2">
         <property name="text">
          <string>Node 2</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="TreeViewComboBox" name="cbNode2"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="labelNode3">
         <property name="text">
          <string>Node 3</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>398</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="5" column="2">
        <widget class="TreeViewComboBox" name="cbNode3"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelNodeHeader">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Triangle nodes</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabProjection">
      <attribute name="title">
       <string>Projection</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
  <customwidget>
   <class>TreeViewComboBox</class>
   <extends>QComboBox</extends>
   <header>commonfrontend/widgets/TreeViewComboBox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>leName</tabstop>
  <tabstop>leComment</tabstop>
  <tabstop>cbDataSource</tabstop>
  <tabstop>cbFileRequester</tabstop>
  <tabstop>cbXCoordinate</tabstop>
  <tabstop>cbYCoordinate</tabstop>
  <tabstop>cbZCoordinate</tabstop>
  <tabstop>cbNode1</tabstop>
  <tabstop>cbNode2</tabstop>
  <tabstop>cbNode3</tabstop>
  <tabstop>cbMatrix</tabstop>
  <tabstop>chkVisible</tabstop>
  <tabstop>sbColorFillingOpacity</tabstop>
  <tabstop>kcbColorFilling</tabstop>
  <tabstop>cbColorFillingMap</tabstop>
  <tabstop>cbColorFillingType</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
